.github/workflows/docker-workflow.yml

```diff
@@ -0,0 +1,262 @@
+name: Build, Deploy and release Docker images
+on:
+  workflow_call:
+    inputs:
+      docker_file:
+        required: true
+        type: string
+        description: path to the Dockerfile used for build step
+      docker_image:
+        required: true
+        type: string
+        description: name of image used for build step and optional push to registries
+      build_args:
+        required: false
+        type: string
+        description: list of arguments used for build step (comma separated)
+      platforms:
+        required: false
+        type: string
+        default: linux/amd64 #,linux/arm/v7
+        description: list of platform to build and optionally push (comma separated)
+      github_repo:
+        required: true
+        type: string
+        description: the git repository that triggered the workflow run
+      github_ref:
```

Comment:
seems to be the same as `version`

Reply:
both version and github_ref inputs were deleted